change/@fluentui-react-ef007003-8eb0-4f9c-9f4d-71de8aa28b6e.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Role prop of commond button can now take tab.role value in Tabs.base and link.role value in Pivot.Base",
```

Since the change files are used to generate release notes, it's best to include the name of the component being modified (and only the changes for this package)
```suggestion
  "comment": "Pivot: role prop of header button can now take link.role value",
```